acceptance/localcluster/localcluster.go

```diff
@@ -332,6 +332,7 @@ func (l *Cluster) processEvent(e dockerclient.EventOrError) bool {
 	// An event on any other container is unexpected. Die.
 	select {
 	case <-l.stopper:
+	case <-monitorStopper:
```

Comment:
l.monitorStopper is set to nil during shutdown. Maybe it shouldn't be, but it's not clear how repeated start and stop calls are supposed to interact.